.deployments/ropsten/ropsten-1658509923.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Hey, is there any way we can document the mainnet deployment in this repo? With the other deploy script we get the .json being saved. Maybe just add a .md file to the .deployments folder with the mainnet info, just so complete and all in one place. Just a thought. Looks great otherwise!

Reply:
I think it's documented right? See file `mainnet-1656441045,json`. The only reason theres a new ropsten file is because I used the test script to make sure deployment still OK after the changes.

Reply:
I am not seeing it in the dif on github or the repo branch itself? Could the push have been reverted due to org name change?

Reply:
I see that file locally, just not the info saved in the file

Reply:
It's here on GitHub - https://github.com/arcadexyz/v2-contracts/blob/main/.deployments/mainnet/mainnet-1656441045.json

So seems like something related to your local?

Reply:
Yeah, I have that file locally, was saying we should add the new ItemsVerifier deployment to it. Not a super big deal though. Sorry I was not explaining very well, we can discuss tomorrow.